common/src/Model/FindGroupVisitor.cpp

```diff
@@ -34,10 +37,34 @@ namespace TrenchBroom {
         void FindGroupVisitor::doVisit(Entity* entity) {}
         void FindGroupVisitor::doVisit(Brush* brush) {}
 
+        // FindOutermostClosedGroupVisitor
+
+        void FindOutermostClosedGroupVisitor::doVisit(World* world) {}
+        void FindOutermostClosedGroupVisitor::doVisit(Layer* layer) {}
+
+        void FindOutermostClosedGroupVisitor::doVisit(Group* group) {
+            const bool closed = !(group->opened() || group->hasOpenedDescendant());
+
+            if (closed) {
+                setResult(group);
+            }
+        }
+
+        void FindOutermostClosedGroupVisitor::doVisit(Entity* entity) {}
+        void FindOutermostClosedGroupVisitor::doVisit(Brush* brush) {}
+
+        // Helper functions
+
         Model::Group* findGroup(Model::Node* node) {
             FindGroupVisitor visitor;
             node->escalate(visitor);
             return visitor.hasResult() ? visitor.result() : nullptr;
         }
+
+        Model::Group* outermostClosedGroup(Model::Node* node) {
```

Comment:
I know I'm nitpicking, sorry. But should this not be called `findOutermostClosedGroup`?

Reply:
no problem, good call!